sr_example/scripts/sr_example/arm_examples/sr_right_arm_follow_waypoints.py

```diff
@@ -0,0 +1,58 @@
+#!/usr/bin/env python
+
+# This example demonstrates some of the functions of the arm commander.
+# The arm is moved through a sequence of goals generated via different functions in the commander.
+
+import rospy
+from trajectory_msgs.msg import JointTrajectory, JointTrajectoryPoint
+from sr_robot_commander.sr_arm_commander import SrArmCommander
+import geometry_msgs.msg
+
+
+rospy.init_node("move_arm_following_waypoitn_trajectory", anonymous=True)
+
+arm_commander = SrArmCommander(set_ground=True)
+
+rospy.sleep(rospy.Duration(1))
+
+# Moving arm to initial pose
+pose_1 = [ 0.32, 0.27, 1.026, 0.0, 0.0, 0.0, 1.0]
+
+print "Moving to initial pose"
+arm_commander.plan_to_pose_target(pose_1)
+arm_commander.execute()
+
+rospy.sleep(rospy.Duration(2))
+
+print "Moving following waypoints trajectory"
```

Comment:
hmm, this statement is a bit confusing, maybe put something like 'Following trajectory defined by waypoints', or something like this?